packages/cloudflare/src/api/overrides/tag-cache/do-sharded-tag-cache.spec.ts

```diff
@@ -1,10 +1,6 @@
 import { afterEach, beforeEach, describe, expect, it, vi } from "vitest";
 
-import doShardedTagCache, {
-  DEFAULT_HARD_REPLICAS,
-  DEFAULT_SOFT_REPLICAS,
-  TagCacheDOId,
-} from "./do-sharded-tag-cache";
+import shardedDOTagCache, { TagCacheDOId } from "./do-sharded-tag-cache";
```

Comment:
should we rename this to DOTagCacheID?

Reply:
What about even simpler `DOId` which I think is what this is? (and it's only exported for tests).

/cc @conico974

Reply:
Ah wasn't aware it was only being exported for tests, in that case i guess it doesn't really matter as much